odl/contrib/datasets/images/cambridge.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Images provided by the University of Cambridge."""
+
+import numpy as np
+from odl.contrib.datasets.util import get_data
+from scipy.misc import imresize
+
+__all__ = ('brain_phantom', 'resolution_phantom', 'building', 'rings',
+           'blurring_kernel')
+
+
+DATA_SUBSET = 'images_cambridge'
+URL_CAM = 'http://store.maths.cam.ac.uk/DAMTP/me404/data_sets/'
+
+
+def convert(image, shape, gray=None, dtype='float64', normalize='max'):
+    """Convert image to standardized format.
+
+    Several properties of the input image may be changed including the shape,
+    data type and maximal value of the image. In addition, this function may
+    convert the image into an ODL object and/or a gray scale image.
+    """
+
+    image.astype(dtype)
+
+    if gray is not None and gray:
+        image = np.mean(image, 2)
+
+    if shape is not None:
+        image = imresize(image, shape)
+
+    if normalize == 'max':
+        image = image / image.max()
+
+    elif normalize == 'sum':
+        image = image / image.sum()
+
+    return image
+
+
+def brain_phantom(shape=None):
+    """Brain phantom for FDG PET simulations.
+
+    Output:
+        image type: gray scales
+        shape: [1024, 1024] (if not specified by `size`)
+        scale: [0, 1]
+        type: float64
```

Comment:
Can you convert that to our Numpydoc style? Same for the other functions

Reply:
What is your Numpydoc style?